src/Silverback.Integration.Kafka/Messaging/Messages/KafkaMessageHeaders.cs

```diff
@@ -18,10 +18,5 @@ public static class KafkaMessageHeaders
         ///     Used to temporary store the target partition index for the produced message.
         /// </summary>
         public const string KafkaPartitionIndex = "x-kafka-partition-index";
-
-        /// <summary>
-        ///     The header that will be filled with the timestamp of the message consumed from Kafka.
-        /// </summary>
-        public const string TimestampKey = "x-kafka-message-timestamp";
     }
```

Comment:
I'd expect `x-source-topic`, `x-source-partition`, `x-source-offset` and other Kafka specific headers to be defined in here (e.g. `x-soruce-consumer` might also make less sense for the other brokers, but maybe we can find other values to use for them...if we don't rename it to something more specific like `x-source-consumer-group`, then in this case it would definitely be Kafka specific).

Reply:
I moved the Kafka specific headers in this class.